pkgs/tools/system/gotop/default.nix

```diff
@@ -14,6 +14,8 @@ buildGoModule rec {
   runVend = true;
   vendorSha256 = "1mbjl7b49naaqkr2j658j17z9ryf5g3x6q34gvmrm7n9y082ggnz";
 
+  buildFlagsArray = [ "-ldflags=-s -w -X main.Version=v${version}" ];
```

Comment:
Why do we also add `-s` and `-w`?

Reply:
This was requested in another PR (#111201). Turning off the debugging information and the stripping saves almost 10 % of the binary size in this case. Sure, `gdb` and other approaches to solve issue will no longer work.